src/Datadog.Trace/Configuration/CsvMapConfigurationSource.cs

```diff
@@ -0,0 +1,60 @@
+using System;
+using System.Collections.Concurrent;
+
+namespace Datadog.Trace.Configuration
+{
+    /// <summary>
+    /// Represents a configuration source that retrieves values from
+    /// a comma-separate list of key-value pairs.
+    /// </summary>
+    public class CsvMapConfigurationSource : StringConfigurationSource
```

Comment:
Is this file used anymore?

Reply:
No, oops, I thought I had deleted it. Good catch.